### ISSUE TITLE
[Bug]: Error Z-3 when launching the .exe

### ISSUE THREAD

**Issue by BlocusEnergy** [ThioJoe/YT-Spammer-Purge#581]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [X] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I've followed all the indications for the client_secrets, but after i connect it whith my google account, when i launch YTspammer he say permission denied (errno 13) and unknown error Code : Z-3

Sorry for bad english.

### Release version

2.14.3

### Steps to reproduce

I don't think there's a manner to reproduce it but 
1. Follow installation tutorial
2. Log your google account with it and relaunch the folder
3. Bug appear

### What platform are you seeing this problem on?

Windows (.exe file)

### Relevant log output

```shell
Error Message: [Errno 13] Permission denied: 'C:\\Users\\nilsb\\Desktop\\YTBSpammer\\SpamPurge_Resources\\Spam_Lists\\SpamDomainsList.txt'

Unknown Error - Code: Z-3 occurred. If this keeps happening, consider posting a bug report
on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report
```

### Screenshots

![unknown](https://user-images.githubusercontent.com/99139228/152694441-0fca1bc2-90a6-4ed4-834a-1a6a96a9d28d.png)

**Comment by Kurdakov** [comment]
it's similar to just closed bug report - try to move to another folder

**Comment by BlocusEnergy** [comment]
> it's similar to just closed bug report - try to move to another folder

Didn't worked, same error

**Comment by Kurdakov** [comment]
anyway for reference you could take a look https://github.com/ThioJoe/YT-Spammer-Purge/issues/580
Could you click with right button Properties then Security tab -on my PC I see the file has all permissions (and also directory with program), could you check also some other files from distribution, what is different, or try to change permissions for file in they are not set as allowed?

**Comment by BlocusEnergy** [comment]
Bug solved with the #580 thank you very much
